src/components/messenger/list/conversation-item/index.tsx

```diff
@@ -111,8 +147,14 @@ export class ConversationItem extends React.Component<Properties> {
           tabIndex={0}
           role='button'
           is-active={isActive}
+          onMouseEnter={this.toggleHover(true)}
+          onMouseLeave={this.toggleHover(false)}
```

Comment:
Is there a reason we can't use CSS to detect the hover? This is a little dicey

Reply:
Updated to use CSS